### ISSUE TITLE
Some questions about using  the medical grayscale dataset

### ISSUE THREAD

**Issue by Zzzzy411** [tensorlayer/srgan#85]
Hi,I use the SRGAN to enhance the resolution of the grayscale medical images.But I got the blurry results.There are some questions about it.
1.The dimensions of the training HR images are 394*394.So I don't use the crop_sub_imgs_fn to crop the original HR images.Does it matter?Is the dimension of the training image too small?
2.I didn't use the vgg loss because the vgg pre-trained model require three channels input images.And this pre-trained model is trained by the natural images. I just use the mse loss and adversarial loss for generator.The quality of the medical images generated by the generator are very low.Can I use the traditional feature extraction methods or unsupervised feature extraction methods to restore high frequency information?
3.About 150 epochs later,the mse loss tend to be stable.The performance of the model can't be improved.The DIV2K dataset require 2000 epochs to generate high quality images.What caused it?The images in my dataset are similar.
4.I use the keras data augmentation methods without rotation to increase the amount of train data.Does it affect the results?
Thanks for your answer.

**Comment by zsdonghao** [comment]
Hi, for medical data, I highly suggest this repo, https://github.com/nebulaV/DAGAN

It is the implementation of "DAGAN: Deep De-Aliasing Generative Adversarial Networks for Fast Compressed Sensing MRI Reconstruction", which just accepted by TMI.

**Comment by phamnam95** [comment]
@Zzzzy411  Did you successfully apply srgan to your dataset? I am doing similar thing so just want to follow up
